starter_code/bin/seeds.js

```diff
@@ -0,0 +1,78 @@
+const mongoose = require('mongoose');
+const Movie = require('../models/Movie');
+
+mongoose.connect("mongodb://localhost/iron-cinema") //aqui tentamos usar o env e não conseguimos
```

Comment:
no arquivo .env tem que ter a linha "**MONGODB_URI=mongodb://...**"
aqui no seed.js a primeira linha tem que ter "**require('dotenv').config()**" 
e depois é a linha abaixo:
```suggestion
mongoose.connect("process.env.MONGODB_URI") //aqui tentamos usar o env e não conseguimos
```